openfisca_aotearoa/variables/acts/parental_leave/parental_leave.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and entitlement system
+from openfisca_aotearoa.entities import Person
+
+
+class parental_leave__is_primary_carer(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Is primary carer"
+    reference = "http://www.legislation.govt.nz/act/public/1987/0129/latest/DLM120458.html?search=qs_act%40bill%40regulation%40deemedreg_parental+leave_resel_25_h&p=1#DLM120458"
+
+
+class parental_leave__applied_for_leave_or_stopped_working(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Has applied for/taken leave or stopped working immediately"
```

Comment:
i think this can be simplified to http://www.legislation.govt.nz/act/public/1987/0129/latest/DLM120458.html#DLM120458